Duplicati/Library/Utility/Utility.cs

```diff
@@ -1572,5 +1569,29 @@ public static string WrapAsCommandLine(IEnumerable<string> args, bool allowEnvEx
         {
             return string.Join(" ", args.Select(x => WrapCommandLineElement(x, allowEnvExpansion)));
         }
+
+        /// <summary>
+        /// Properly parses commandline args from Environment.CommandLine, keeps all double quotes inplace
+        /// </summary>
+        /// <returns>Array of commandline args.</returns>
+        /// <param name="commandLine">The argument to parse. Environment.CommandLine is ideal.</param>
+        public static IEnumerable<string> ProperParseArgs(string commandLine)
+        {
+            var argsBuilder = new StringBuilder(commandLine);
+            var inQuote = false;
+
+            // Convert the spaces to a newline sign so we can split at newline later on
+            // Only convert spaces which are outside the boundries of quoted text
+            for (var i = 0; i < argsBuilder.Length; i++)
+                if (argsBuilder[i] == '"')
+                    inQuote = !inQuote;
+                else if (argsBuilder[i] == ' ' && !inQuote)

```

Comment:
Arguments can be separated by multiple whitespace characters, including tabs, so this does not work.